scss/_base/_forms.scss

```diff
@@ -1,59 +1,99 @@
+// A label and a form element.
+//
+// .-padded - Adds extra padding above and below field group.
+// .-inline - Form elements appear side-by-side.
+//
+// Styleguide Forms - Field Groups
+.field-group {
+  margin-bottom: ($base-spacing / 2);
```

Comment:
gotcha, ok. Would it still be better to do `margin-top` instead so that we don't get weird bottom spacing to adjust?